app/main.py

```diff
@@ -1,3 +1,111 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import math
+
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def get_dict_from_json() -> dict:
+    with open("app/config.json", "r") as data:
+        data = json.load(data)
+
+    customers = [Customer(
+        name=customer["name"],
+        product_cart=customer["product_cart"],
+        location=customer["location"],
+        money=customer["money"],
+        car=Car(brand=customer["car"]["brand"],
+                fuel_consumption=customer["car"]["fuel_consumption"], )
+    ) for customer in data["customers"]]
+
+    shops = [Shop(
+        name=shop["name"],
+        products=shop["products"],
+        location=shop["location"],
+    ) for shop in data["shops"]]
+
+    return {
+        "FUEL_PRICE": data["FUEL_PRICE"],
+        "customers": customers,
+        "shops": shops,
+    }
+
+
+def length_trip_calculation(x_point: list, y_point: list) -> float:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]